utility_analysis_new/combiners.py

```diff
@@ -158,49 +142,65 @@ def compute_pmf(self) -> np.ndarray:
         return pmf
 
 
-class PartitionSelectionCombiner(pipeline_dp.Combiner):
+# PartitionSelectionAccumulator = (probabilities, moments). They are mutually
```

Comment:
nit: Maybe `These two`/`The two elements` instead of `They` is more descriptive, I could see someone being confused about what `They` refers to here.